cmake/sdlchecks.cmake

```diff
@@ -1,17 +1,25 @@
 include(CMakeParseArguments)
+include(${SDL2_SOURCE_DIR}/cmake/sdlfind.cmake)
```

Comment:
I would just include it in the main `CMakeLists.txt`, where we include other files.